frontend/cypress/integration/common/graph_display.ts

```diff
@@ -0,0 +1,365 @@
+  input.should('exist');
+  input.should('not.be.checked');
+  input = cy.get('button#display-settings').get(`input#trafficRate`);
+  input.should('exist');
+  input.should('not.be.checked');
+  input = cy.get('button#display-settings').get(`input#boxByCluster`);
+  input.should('exist');
+  input.should('be.checked');
+  input = cy.get('button#display-settings').get(`input#boxByNamespace`);
+  input.should('exist');
+  input.should('be.checked');
+  input = cy.get('button#display-settings').get(`input#filterHide`);
+  input.should('exist');
+  input.should('be.checked');
+  input = cy.get('button#display-settings').get(`input#filterIdleEdges`);
+  input.should('exist');
+  input.should('not.be.checked');
+  input = cy.get('button#display-settings').get(`input#filterIdleNodes`);
+  input.should('exist');
+  input.should('not.be.checked');
+  input = cy.get('button#display-settings').get(`input#filterOperationNodes`);
+  input.should('exist');
+  input.should('not.be.checked');
+  input = cy.get('button#display-settings').get(`input#rank`);
+  input.should('exist');
+  input.should('not.be.checked');
+  input = cy.get('button#display-settings').get(`input#filterServiceNodes`);
+  input.should('exist');
+  input.should('be.checked');
+  input = cy.get('button#display-settings').get(`input#filterTrafficAnimation`);
+  input.should('exist');
+  input.should('not.be.checked');
+  input = cy.get('button#display-settings').get(`input#filterSidecars`);
+  input.should('exist');
+  input.should('be.checked');
+  input = cy.get('button#display-settings').get(`input#filterSecurity`);
+  input.should('exist');
+  input.should('not.be.checked');
+  input = cy.get('button#display-settings').get(`input#filterVS`);
+  input.should('exist');
+  input.should('be.checked');
+});
+
+Then('the graph reflects default settings', () => {
+  cy.waitForReact(1000, '#root');
+  cy.getReact('CytoscapeGraph')
+    .getCurrentState()
+    .then(state => {
+      // no nonDefault edge label info
+      let numEdges = state.cy.edges(`[?responseTime],[?throughput]`).length;
+      assert.isTrue(numEdges === 0);
+
+      // no idle edges, mtls
+      numEdges = state.cy.edges(`[^hasTraffic],[isMTLS > 0]`).length;
+      assert.isTrue(numEdges === 0);
+
+      // boxes
+      let numNodes = state.cy.nodes(`[isBox = "app"]`).length;
+      assert.isTrue(numNodes > 0);
+      numNodes = state.cy.nodes(`[isBox = "namespace"]`).length;
+      assert.isTrue(numNodes > 0);
+
+      // service nodes
+      numNodes = state.cy.nodes(`[nodeType = "service"]`).length;
+      assert.isTrue(numNodes > 0);
+
+      // a variety of not-found tests
+      numNodes = state.cy.nodes(
+        `[isBox = "cluster"],[?isIdle],[?rank],[?hasMissingSC],[?hasVS],[nodeType = "operation"]`
+      ).length;
+      assert.isTrue(numNodes === 0);
+    });
+});
+
+Then('user sees {string} edge labels', el => {
+  const input = cy.get('button#display-settings').get(`input#${el}`);
```

Comment:
Same comment here and further down about assigning to return values.

Reply:
Ok, I believe you, but I do still have the same question, what's the problem :)